.env.development

```diff
@@ -0,0 +1,2 @@
+SESSION_SECRET=&PX$fuY@&7E&UMTnFB%Vqd8REFpTbHm5rbKh^A%&4s#34+sY7g*5kv#Z!C_*NGW3
```

Comment:
This is just for testing. We will need to define the same ENV vars in production